### ISSUE TITLE
The history table is not created using semantic HTML for a table

### ISSUE THREAD

**Issue by Tanvez** [Harvard-University-iCommons/chainlit#26]
| IMPACT      | URL          | ISSUE DESCRIPTION   | WCAG CRITERIA | TESTING METHOD | SCREENSHOT or SNIPPET | ACTION |
| :----:          |    :----:      |      :----:                        |  :----:                    | :----:                        | :----:                                     |  :----:    | 
| High    | n/a      | The table is not created susing semantic HTML for a table but instead uses div and paragraph tags. Consequently what we see as a table, and how we understand the rows and columns are associated with each other is not conveyed programatically for assistive technlogy.   | 1.3.1 Info and Relationships - A | Manual - Safari/VO | [link to the screenshot](https://share.getcloudapp.com/xQuE8WB1) | For assistive technology to correctly convey the relationship of the column headers to the cells of information they describe and the realtionship between information presented visually, the HTML <table> element should be used to markup this content.
